tcms/testruns/templates/testruns/get.html

```diff
@@ -82,6 +82,10 @@ <h2 class="card-pf-title" style="text-align: left">
     {% include 'include/tags_card.html' with add_perm=perms.testruns.add_testruntag %}
 </div>
 
+<div class="col-xs-12 col-sm-6 col-md-6">
+    {% include 'include/attachments.html' %}
+</div>
+
```

Comment:
This block and the other cards must go inside the parent `<div class="container-cards-pf">`